File: src/pkg01_helloworld_cpp/src/helloworld.cpp
#include <rclcpp/rclcpp.hpp>

class MyNode:public rclcpp::Node{ 
public:
    MyNode(const std::string &node_name):
        Node(node_name){
            RCLCPP_INFO(this->get_logger(),
                "helloworld");
        }
};

int main(int argc, char const *argv[])
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<MyNode>("cpp_node");
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: src/pkg01_helloworld_py/pkg01_helloworld_py/helloworld.py
import rclpy
from rclpy.node import Node

class MyNode(Node):
    def __init__(self,node_name):
        super().__init__(node_name)
        self.get_logger().info("helloworld")
        self.get_logger().error("helloworld")
        
def main():
    rclpy.init()
    node = MyNode("py_node")
    rclpy.spin(node)
    rclpy.shutdown()
    
